app/serializers/player_serializer.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import serializers
+from app.models import Player
+from django.db import models
+from enumfields import EnumField, Enum
+from app.models.player import PlayerRole
+from app.serializers.faction_serializer import FactionSerializer
+from app.serializers.game_serializer import GameSerializer
+
+class PlayerSerializer(serializers.ModelSerializer):
+    roleChar = serializers.SerializerMethodField('getRole')
+
+    def getRole(self, player):
+        return player.role.value
+
+    name = serializers.SerializerMethodField('getName')
+
+    def getName(self, player):
+        return player.user.get_full_name()
+
+    email = serializers.SerializerMethodField('getEmail')
+
+    def getEmail(self, player):
+        return player.user.email
+
+    faction = FactionSerializer()
+    game = GameSerializer()
+    
+    class Meta:
+        model = Player
+        fields = ['code', 
+            'roleChar', 
+            'is_oz',
+            'name',
+            'email',
+            'score',
+            'shop_score',
+            'active',
+            'game',
+            'faction']
```

Comment:
We shouldn't have fields like `active` here, as we should only be returning active players anyway.

Reply:
That's fair.